src/test/skript/tests/syntaxes/expressions/ExprLootContext.sk

```diff
@@ -0,0 +1,41 @@
+test "loot context":
+
+	set {_context} to a new loot context at spawn of "world"
```

Comment:
all the spawn of "world" should be replaced with `test-location` (or `test location`!) :)